### DISCUSSION TITLE
2024/04/07/ops/migrating-docker-images/

### DISCUSSION THREAD

**Discussion by giscus** [s1eke/s1eke.github.io#1] [Announcements]
# 2024/04/07/ops/migrating-docker-images/

迁移镜像有三种场景 ：  在线迁移：通常是有一台机器能同时连通源镜像仓库和目标镜像仓库就算在线迁移 离线迁移：只要源镜像仓库和目标镜像仓库无法在一台机器上同时联通就算离线迁移 备份：没错，备份也是一种迁移  前置准备迁移之前首先需要有一份迁移的目标，这里我会以部署 k8s 所需要的容器为例。首先需要获取一份所有的镜像清单 images.list，这里通过 kubespray 的离线脚本自动生成。

https://blog.jugg.xyz/2024/04/07/ops/migrating-docker-images/

**Comment by s1eke** [comment]
# test
## test
test
